data/competencies.yml

```diff
@@ -1188,3 +1188,16 @@
   level: senior2
   area: leadership
   domain: null
+
+# S2
+- id:
+  summary: Actively encourages or shapes the personal development of more junior people
```

Comment:
I was looking for a different way to say "more junior people", because I see the word junior more as a noun describing the first step of a role than an adjective with gradients. 

For instance I guess you can say that a Principal is more junior than a Tech Director but since junior engineer is also a role within that scale, it's not the clearest way to express that. So I guess you could say "less experienced people"? But upon reflection specifying might be unnecessary. You can play that role for a peer, for instance as a tech lead or mentor to someone who is also an S2 but has a different set of skills.

```
Actively encourages or shapes the personal development of others
```

Reply:
Love this suggestion, thank you 👍